Timber LumberYard exception

```
Caused by: java.lang.NullPointerException: Attempt to get length of null array
   at io.palaima.debugdrawer.timber.data.LumberYard.cleanUp(LumberYard.java:140)
```

--- a/debugdrawer-timber/src/main/java/io/palaima/debugdrawer/timber/data/LumberYard.java
+++ b/debugdrawer-timber/src/main/java/io/palaima/debugdrawer/timber/data/LumberYard.java
@@ -137,9 +137,11 @@ public class LumberYard {
         File dir = getLogDir();
         if (dir != null) {
             File[] files = dir.listFiles();
-            for (File file : files) {
-                if (file.getName().endsWith(LOG_FILE_END)) {
-                    file.delete();
+            if (files != null) {
+                for (File file : files) {
+                    if (file.getName().endsWith(LOG_FILE_END)) {
+                        file.delete();
+                    }
                 }
             }
         }

